python/ray/autoscaler/command_runner.py

```diff
@@ -706,6 +706,10 @@ def _docker_expand_user(self, string, any_char=False):
         return string
 
     def run_init(self, *, as_head, file_mounts):
+        BOOTSTRAP_MOUNTS = [
```

Comment:
I don't think you need these? I believe `commands.py` is responsible for adding these as file mounts.